docs/getting-started/configure-integration-policy.asciidoc

```diff
@@ -68,6 +68,16 @@ Malware protection also allows you to manage a blocklist to prevent specified ap
 extending the list of processes that {elastic-defend} considers malicious. Use the **Blocklist enabled** toggle
 to enable or disable this feature for all hosts associated with the integration policy. To configure the blocklist, refer to <<blocklist>>.
 
+When *Detect* malware is enabled, {agent} will quarantine any malicious file it finds. Specifically {agent} will remove the file from its current location, obfuscate it (XOR encrypt it with the encryption key `ELASTIC`), move it to a different folder, and rename it as a string like `318e70c2-af9b-4c3a-939d-11410b9a112c`.
+
+- On macOS, all {agent} versions move quarantined files to the folder `/System/Volumes/Data/.equarantine`.
+- On Linux, all {agent} versions move quarantined files to the folder `.equarantine` at the root of the mount point of the file being quarantined.
+- On Windows, {agent} versions 8.5 and later move quarantined files to the folder `[DriveLetter:]\.quarantine`, unless they are from the `C:` drive. These files are moved to `C:\Program Files\Elastic\Endpoint\state\.equarantine`. 
++
+{agent} versions earlier than 8.5 move quarantined files to `[DriveLetter:]\.quarantine`, regardless of their original drive.
```

Comment:
A lot of repetition here, mostly of phrasing that feels like filler. A table might be cleaner, or reformat the list to focus on essentials (you might still need to add a lead-in for a list like this):
```suggestion
- macOS: `/System/Volumes/Data/.equarantine`
- Linux: `.equarantine` at the root of the mount point of the file being quarantined
- Windows - {agent} versions 8.5 and later: `[DriveLetter:]\.quarantine`, unless the files are from the `C:` drive. These files are moved to `C:\Program Files\Elastic\Endpoint\state\.equarantine`.
- Windows - {agent} versions 8.4 and earlier: `[DriveLetter:]\.quarantine`, regardless of the files' original drive
```